src/main/java/org/jabref/gui/preferences/EntryEditorTabViewModel.java

```diff
@@ -78,35 +82,37 @@ public void setValues() {
 
     @Override
     public void storeSettings() {
-        preferences.putBoolean(JabRefPreferences.AUTO_OPEN_FORM, openOnNewEntryProperty.getValue());
-        preferences.putBoolean(JabRefPreferences.DEFAULT_SHOW_SOURCE, defaultSourceProperty.getValue());
-        preferences.putBoolean(JabRefPreferences.SHOW_RECOMMENDATIONS, enableRelatedArticlesTabProperty.getValue());
-        preferences.putBoolean(JabRefPreferences.ACCEPT_RECOMMENDATIONS, acceptRecommendationsProperty.getValue());
-        preferences.putBoolean(JabRefPreferences.SHOW_LATEX_CITATIONS, enableLatexCitationsTabProperty.getValue());
-        preferences.putBoolean(JabRefPreferences.VALIDATE_IN_ENTRY_EDITOR, enableValidationProperty.getValue());
-
-        autoCompletePreferences.setShouldAutoComplete(enableAutoCompleteProperty.getValue());
-        autoCompletePreferences.setCompleteNames(autoCompleteFieldsProperty.getValue());
-        if (autoCompleteBothProperty.getValue()) {
-            autoCompletePreferences.setOnlyCompleteFirstLast(false);
-            autoCompletePreferences.setOnlyCompleteLastFirst(false);
-        } else if (autoCompleteFirstLastProperty.getValue()) {
-            autoCompletePreferences.setOnlyCompleteFirstLast(true);
-            autoCompletePreferences.setOnlyCompleteLastFirst(false);
-        } else {
-            autoCompletePreferences.setOnlyCompleteFirstLast(false);
-            autoCompletePreferences.setOnlyCompleteLastFirst(true);
+        preferencesService.storeEntryEditorPreferences(new EntryEditorPreferences(
```

Comment:
Can these constants `JabRefPreferences.AUTO_OPEN_FORM` etc now be made private?

Reply:
Sadly no, not yet, since they are still in use outside the preferences dialog. I did not yet remove all the calls to JabRefPreferences::get and ::put in the whole codebase, since this would make this PR completly unreviewable.
I wanted to do this step by step.